Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -4,7 +4,7 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 
-Test-writing utilities that simplify creating O3DE in-Editor tests in Python.
+Test-writing utilities that simplify creating O3DE Editor tests in Python.
```

Comment:
please revert this comment change or clarify it further: it's possible to test parts of the editor without also running inside of it

Reply:
reverted in next push.